### DISCUSSION TITLE
Where i can download the config for adapters?

### DISCUSSION THREAD

**Discussion by tonymacx86PRO** [Mikubill/sd-webui-controlnet#544] [Q&A]
My style adapter seems to not work because i don't downloaded configs
Where i can download it for tencent adapters models?

**Comment by brunogcar** [comment]
does this https://github.com/Mikubill/sd-webui-controlnet/issues/547#issuecomment-1461050662 solve your problem?

yaml files are in https://github.com/Mikubill/sd-webui-controlnet/tree/main/models, should be downloaded with the extension or when you update it
